azurerm/resource_arm_recovery_services_vault_test.go

```diff
@@ -43,6 +41,39 @@ func TestAccAzureRMRecoveryServicesVault_basic(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMRecoveryServicesVault_requiresImport(t *testing.T) {
+	if !requireResourcesToBeImported {
+		t.Skip("Skipping since resources aren't required to be imported")
+		return
+	}
+
+	ri := tf.AccRandTimeInt()
+	resourceName := "azurerm_recovery_services_vault.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMRecoveryServicesVaultDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMRecoveryServicesVault_basic(ri, testLocation()),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMRecoveryServicesVaultExists(resourceName),
+					resource.TestCheckResourceAttrSet(resourceName, "name"),
+					resource.TestCheckResourceAttrSet(resourceName, "location"),
+					resource.TestCheckResourceAttrSet(resourceName, "resource_group_name"),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
+					resource.TestCheckResourceAttr(resourceName, "sku", string(recoveryservices.Standard)),
```

Comment:
IME it's better to hard-code these values in the tests rather than reusing the enum, since if the value changes in the Swagger/SDK (e.g. from Title -> lower case) then we miss a breaking change here